lighthouse-core/test/gather/driver-test.js

```diff
@@ -22,7 +22,10 @@ const redirectDevtoolsLog = require('../fixtures/wikipedia-redirect.devtoolslog.
 const MAX_WAIT_FOR_PROTOCOL = 20;
 
 function createOnceStub(events) {
-  return (eventName, cb) => {
+  return async (eventName, cb) => {
```

Comment:
this is a weird use of async/await (and depending on `driver.once` calls you'll end up with an unhandled promise rejection for the throw below).

Can we just
```js
// wait a tick b/c real events never fire immediately
setTimeout(_ => cb(events[eventName]), 0);
return;
```
like normal people? :P

Reply:
I meant drop the async/await in favor of the setTimeout :)